docs/serving/samples/hello-world/helloworld-csharp/README.md

```diff
@@ -87,6 +87,15 @@ cd knative-docs/serving/samples/hello-world/helloworld-csharp
    CMD ["dotnet", "out/helloworld-csharp.dll"]
    ```
 
+1. Add a `.dockerignore` file to ensure local builds cannot disrupt container builds.
```

Comment:
Why tell folks to add a `.dockerignore` file since you added it already. If they clone the source code they will have the file already there.

Reply:
We don't tell them to clone the source code, we tell them to assemble the files through CLI scaffolding and copy & paste.

Reply:
```suggestion
1. Create a `.dockerignore` file to ensure that any files related to a local build, do not affect the container that you build for deployment.
```

Can we make this change throughout?